package.json

```diff
@@ -2219,7 +2232,8 @@
             {
                 "command": "vscode-docker.containers.openFile",
                 "title": "%vscode-docker.commands.containers.openFile%",
-                "category": "%vscode-docker.commands.category.dockerContainers%"
+                "category": "%vscode-docker.commands.category.dockerContainers%",
+                "icon": "$(file-symlink-file)"
```

Comment:
```suggestion
                "icon": "$(go-to-file)"
```

`go-to-file` is used in the source control pane for opening a file directly, I'd suggest that over `file-symlink-file`.
![image](https://user-images.githubusercontent.com/36966225/105505826-69d02d80-5c97-11eb-9a69-efcef8bd1a95.png)

Reply:
Yes, I like that one better.  I didn't come across it when searching.

<img width="517" alt="Screenshot 2021-01-25 at 10 23 25" src="https://user-images.githubusercontent.com/6402946/105748850-879bcd80-5ef7-11eb-9b82-aadaaa2b953b.png">